src/dotty/tools/backend/jvm/DottyBackendInterface.scala

```diff
@@ -944,7 +944,7 @@ class DottyBackendInterface(outputDirectory: AbstractFile, val superCallsMap: Ma
   }
 
   object ValDef extends ValDefDeconstructor {
-    def _1: Modifiers = field.mods
+    def _1: Modifiers = tpd.Modifiers(field.symbol)
```

Comment:
Grepping the source code it seems that _1 is never used. Try returning `null` to ensure it?